src/forwarders.js

```diff
@@ -589,33 +604,78 @@ export default function Forwarders(mpInstance, kitBlocker) {
                 'No config was passed. Cannot process forwarders'
             );
         } else {
-            try {
-                if (
-                    Array.isArray(config.kitConfigs) &&
-                    config.kitConfigs.length
-                ) {
-                    config.kitConfigs.forEach(function(kitConfig) {
-                        self.configureForwarder(kitConfig);
-                    });
-                }
+            this.processMpConfiguredKits(config);
+            this.processSideloadedKits(config);
+            this.processPixelConfigs(config);
 
-                if (
-                    Array.isArray(config.pixelConfigs) &&
-                    config.pixelConfigs.length
-                ) {
-                    config.pixelConfigs.forEach(function(pixelConfig) {
-                        self.configurePixel(pixelConfig);
-                    });
+            self.initForwarders(
+                mpInstance._Store.SDKConfig.identifyRequest.userIdentities,
+                forwardingStatsCallback
+            );
+        }
+    };
+
+    this.processMpConfiguredKits = function(config) {
```

Comment:
I think `processImportedKits` or `processConfiguredKits` sounds more appropriate to me, as they are imported from the config, mpConfigured kits sounds vague.

If the intention is to denote that they came from the mparticle config, then perhaps `processKitsFromMPConfig` might be more appropriate.

Reply:
hm, so there is the rare use case where a company is npm installing the kits, but is overriding the config settings with their own.  so really this is for any kit configured from our UI.  i'll do `processUIConfiguredKits`

Reply:
or I'll do `processUIEnabledKits` actually